### PR TITLE
Update cat file transformer with chop lump dunk [beta edition]

### PR DESCRIPTION
**Pull request by yaoandrew**
This is a new PR which modifies the existing cat file chop lump transformer to save `cat1.1.0` dunk events.

### CODE REVIEW

File: db/migrations/20180912171047_create_cat_file.sql [hunk 1]
```diff
@@ -1,21 +1,23 @@
 -- +goose Up
-CREATE TABLE maker.cat_file_chop_lump
+CREATE TABLE maker.cat_file_chop_lump_dunk
```
Comment (rmulhol) [inline db/migrations/20180912171047_create_cat_file.sql]: unfortunately, we can't modify an earlier migration if we're looking to have this applied to `beta`. This should work for `staging` though 👍

**Review by rmulhol — CHANGES_REQUESTED** [review]
Needs to be addressed via backwards-compatible changes (new migration and/or leaving the table name unchanged [leaving the table unchanged actually isn't that bad of an option since we can always alias in postgraphile])

File: transformers/component_tests/queries/all_ilk_file_events_query_test.go [hunk 2]
```diff
@@ -49,17 +50,24 @@ var _ = Describe("Ilk File Events Query", func() {
 		timestampOne = int(rand.Int31())
 		headerOne = createHeader(blockOne, timestampOne, headerRepo)
 		logOneId = test_data.CreateTestLog(headerOne.Id, db).ID
+		msgSender = test_data.Cat110Address()
```
Comment (rmulhol) [inline transformers/component_tests/queries/all_ilk_file_events_query_test.go]: I wonder if we might want to use a different (potentially random) address here - the contract itself is unlikely to be the `msg.sender`, and I could see some tricky-to-debug test situations if the contract address and message sender IDs are the same

File: db/migrations/20180912171047_create_cat_file.sql (hunk 1 above)
Comment (yaoandrew) [reply db/migrations/20180912171047_create_cat_file.sql]: Oh cool. I didn't realize we could alias it in postgraphile. That sounds like a better bet.

File: transformers/component_tests/queries/all_ilk_file_events_query_test.go (hunk 2 above)
Comment (yaoandrew) [reply transformers/component_tests/queries/all_ilk_file_events_query_test.go]: Yeah.. makes sense. I think I spaced out on assigning the contract address as the msgSender.

File: transformers/events/cat_file/chop_lump_dunk/transformer_test.go [hunk 3]
```diff
@@ -44,8 +45,7 @@ var _ = Describe("Cat file chop lump transformer", func() {
 			models, err := transformer.ToModels(constants.Cat100ABI(), []core.EventLog{test_data.CatFileChopEventLog}, db)
 			Expect(err).NotTo(HaveOccurred())
 
-			var ilkID int64
-			ilkErr := db.Get(&ilkID, `SELECT id FROM maker.ilks where ilk = $1`, test_data.CatFileChopEventLog.Log.Topics[2].Hex())
+			ilkID, ilkErr := shared.GetOrCreateIlk(test_data.CatFileChopEventLog.Log.Topics[2].Hex(), db)
```
Comment (elizabethengelman) [inline transformers/events/cat_file/chop_lump_dunk/transformer_test.go:48]: 👏

**Review by elizabethengelman — APPROVED** [review]

File: transformers/shared/constants/method.go [hunk 4]
```diff
@@ -102,6 +102,9 @@ func catClawMethod() string     { return getSolidityFunctionSignature(Cat110ABI(
 func catFileBoxMethod() string {
 	return getOverloadedFunctionSignature(Cat110ABI(), "file", []string{"bytes32", "uint256"})
 }
+func catFileChopDunkMethod() string {
```
Comment (rmulhol) [inline transformers/shared/constants/method.go]: Seems like we could just rename the `catFileChopLumpMethod()` to `catFileChopLumpDunkMethod()`?

**Review by rmulhol — APPROVED** [review]
Would prefer to remove the `CatFileChopDunkSignature()` method + private functions it calls (and get rid of the test) since I think we can use/rename `CatFileChopLumpSignature()` - otherwise LGTM 👍

**Pull request merged**